communication/slack-config/usergroups.yaml

```diff
@@ -4,6 +4,22 @@
 # e.g., @release-managers is defined in sig-release/usergroups.yaml
 
 usergroups:
+  - name: apac-coordinator
```

Comment:
Should this be "apac-coordinators"? Since the team name is sig-contributor-experience-apac-coordinators. I am ok either way, just wanted to check if that matters!

Reply:
yeah that's better!